src/main/java/com/vladbudan/app/dto/modelDto/CatDto.java

```diff
@@ -0,0 +1,19 @@
+package com.vladbudan.app.dto.modelDto;
+
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+
+@Data
+public class CatDto {
+
+    private Long id;
+
+    @NotNull(message = "Name cannot be null")
```

Comment:
Does this annotation work? If yes, share me the screenshot with failed validation